packages/cardpay-cli/testing/create-account.ts

```diff
@@ -0,0 +1,270 @@
+    }
+
+    const createdMnemonic = generateMnemonic();
+
+    const web3OptsL1 = getConnectionType(args);
+    const createdWeb3OptsL2: Web3OptsMnemonic = {
+      connectionType: 'mnemonic',
+      mnemonic: createdMnemonic,
+    };
+    const networkL2 = (L2_NETWORK_BY_L1 as any)[networkL1];
+    if (!networkL2) {
+      throw new Error(`No corresponding l2 network found of l1 network ${networkL1}`);
+    }
+    const { web3: createdWeb3L2 } = await getEthereumClients(networkL2, createdWeb3OptsL2);
+
+    const createdOwner = (await createdWeb3L2.eth.getAccounts())[0];
+
+    const prepaidCardAddress = createPrepaidCardSafe
+      ? await provisionPrepaidCard(createdOwner, networkL2, createdWeb3OptsL2)
+      : undefined;
+    const merchantSafeAddress =
+      prepaidCardAddress && createMerchantSafe
+        ? await registerMerchant(prepaidCardAddress, networkL2, createdWeb3OptsL2)
+        : undefined;
+    const rewardSafeAddress =
+      prepaidCardAddress && createRewardSafe
+        ? await registerRewardee(prepaidCardAddress, networkL2, createdWeb3OptsL2)
+        : undefined;
+
+    const depotAddress = createDepotSafe
+      ? await bridgeToken(
+          createdOwner,
+          networkL1,
+          web3OptsL1,
+          networkL2,
+          createdWeb3OptsL2,
+          bridgeTokenAddress,
+          bridgeTokenAmount
+        )
+      : undefined;
+
+    console.log(`
+
+  WARNING: NOT A SECURE MNEMONIC. ONLY INTENDED FOR TESTING!
+
+      Mnemonic: ${createdMnemonic} 
+      Address: ${createdOwner}
+      Prepaid card: ${prepaidCardAddress ?? 'No prepaid card created'}
+      Merchant safe: ${merchantSafeAddress ?? 'No merchant safe created'}
+      Reward safe: ${rewardSafeAddress ?? 'No reward safe created'}
+      Depot safe: ${depotAddress ?? 'No depot safe created'}
+    `);
+  },
+} as CommandModule;
+
+const provisionPrepaidCard = async (createdOwner: string, network: string, web3Opts: Web3Opts): Promise<string> => {
+  console.log('Creating prepaid card');
+  const { web3 } = await getEthereumClients(network, web3Opts);
+  const provisionerSecret = process.env.PROVISIONER_SECRET;
+  const sku: string = (L2_NETWORK_CONFIG as any)[network].sku;
+  const environment: string = (L2_NETWORK_CONFIG as any)[network].environment;
+  if (!provisionerSecret) {
+    throw new Error('PROVISIONER_SECRET env not provided');
+  }
+  if (!sku) {
+    throw new Error('Default sku does not exist');
+  }
+  if (!environment) {
+    throw new Error('Default environment does not exist');
+  }
+  let prepaidCardMarketV2 = await getSDK('PrepaidCardMarketV2', web3);
+  const prepaidCardMgr = await getSDK('PrepaidCard', web3);
+  let blockExplorer = await getConstant('blockExplorer', web3);
+
+  console.log(
+    `Provisioning a prepaid card from the SKU ${sku} to the EOA ${createdOwner} in the ${environment} environment...`
+  );
+
+  let quantity = await prepaidCardMarketV2.getQuantity(sku);
+
+  if (quantity === 0) {
```

Comment:
Will this comparison work given it says here the quantity is returned as string? https://github.com/cardstack/cardstack/blob/main/packages/hub/node-tests/routes/email-card-drop-request-test.ts#L54

Reply:
Good catch!